templates/docs/patterns/navigation.md

```diff
@@ -89,6 +91,40 @@ To add icons on the left side of the items in side navigation use the `.p-side-n
 View example of the side navigation pattern with icons
 </a></div>
 
+#### Raw HTML
+
+When HTML markup of side navigation links is generated by external service and Vanilla class names cannot be added to the elements a `p-side-navigation--raw-html` modifier can be used.
+
+In this variant side navigation pattern will style headings (`h2`, `h3`, etc.) and lists (`ul`, `li`) with links or text elements (`a`, `strong`, `span`) consistently with default side navigation component:
+
+- headings (`h2`, `h3`, etc.) get the styling of `p-side-navigation__item--title`
+- lists `ul` is styled as `p-side-navigation__list`, and `li` as `p-side-navigation__item`
```

Comment:
```suggestion
-  lists: `ul` tags are styled as `p-side-navigation__list`, and `li` tags as `p-side-navigation__item`
```

Reply:
@lyubomir-popov If we want to add colon to `lists:` should we do it for all the items in the listing for consistency? (`headings:`, `text elements:`, `links:`)?